config/base/dex/numaflow-dex-server-configmap.yaml

```diff
@@ -4,30 +4,29 @@ metadata:
   name: numaflow-dex-server
 data:
   config.yaml: |
-    issuer: <HOSTNAME>/dex
+    issuer: https://localhost:8443/dex
```

Comment:
same as `Homepage URL` of my auth application: https://github.com/settings/applications/2368167